packages/@okta/vuepress-site/docs/guides/terraform-organize-configuration/main/index.md

```diff
@@ -0,0 +1,289 @@
+
+* Create reusable code with modules
+
+* Store secrets securely
+
+* Avoid problems related to configuration drift
+
+#### What you need
+
+* Familiarity with the Terraform terms: configuration, resources, state, and commands. See the [Terraform overview](/docs/guides/terraform-overview).
+
+* An Okta organization.
+
+* A [Terraform configuration](/docs/guides/terraform-enable-org-access/main) that can access your Okta org.
+
+* An Okta user account with the Super Administrator role.
+
+## Overview
+
+Maintaining your Terraform configuration can become more challenging as the amount of code increases. Follow the best practices in this article to create more concise code that's easier to understand and maintain, reduces the risk of errors, and increases security.
+
+## Organize files by resource type and purpose
+
+To increase the maintainability of your Terraform configuration, instead of using one file, use one Terraform script file for each resource type or related purpose. For example, define device assurance policies in a file called `Devices.tf`.
+
+Some benefits of using multiple files for your Terraform configuration are:
+
+* **Industry best practices support maintainability.** Use Terraform industry standards to ensure your Terraform files are easy for you and others to understand and maintain.
+
+* **Support separation by personnel role.** Team members may create and maintain different parts as the configuration becomes more complex. For example, one person may manage group membership while another updates security policies. Separating files by resource type and purpose reduces the risk of accidental changes to unrelated resources.
+
+* **Reduce error risks.** A maintainable configuration decreases the risk of errors, which lowers the chance of security breaches or other problems.
+
+* **Clean version control history.** Organizing your Terraform files in a version control system (such as GitHub) makes it easier to see the modification history for each resource type.
+
+## Modularize files for reuse and maintainability
+
+Use [Terraform resource modules](https://developer.hashicorp.com/terraform/language/modules/syntax) to group related resources and facilitate reuse. Design self-contained modules that focus on a single task or area. For example, when multiple Okta orgs use the same corporate branding assets, you can create a shared module that contains those assets. See the [Terraform article on developing modules](https://developer.hashicorp.com/terraform/language/modules/develop).
+
+You can write one script that works in multiple places by using input and output variables. For example, a script for adding groups can use a variable for the list of group names. You can also use variables to create shared modules for different Terraform configurations. See the [Terraform article on input and output variables](https://developer.hashicorp.com/terraform/language/values/variables).
```

Comment:
I'm assuming it's both. Instead of removing "input and output", I changed the link to:
`[Terraform section on input and output variables](https://developer.hashicorp.com/terraform/language/values)`
I think that fits the intent.

Reply:
In the next commit